.circleci/config.yml

```diff
@@ -36,13 +36,13 @@ aliases:
     executor:
       name: android/android-machine
       resource-class: large
-      tag: 2024.01.1
+      tag: 2024.04.1
 
 version: 2.1
 orbs:
-  android: circleci/android@2.3.0
+  android: circleci/android@2.5.0
   gcp-cli: circleci/gcp-cli@2.1.0
```

Comment:
There's an update for this one too, but didn't change it to minimize the changeset:
- minor: 2.4.1
- latest: 3.2.1